### PR TITLE
Add example for camera_position property

### PR DESCRIPTION
**Pull request by ENIAC-6**
Add example for camera_position property per issue #1629.

### Overview

Added an example for camera_position property to docstring.
Issue #1629 

### Details

Modified example from SciPy2022 tutorial, lesson 02_mesh/a_lesson_mesh.ipynb

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -1308,19 +1308,47 @@ def scale(self, scale):
 
     @property
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
         return self.renderer.camera_position
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1312]:
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 2]
```diff
@@ -1308,19 +1308,47 @@ def scale(self, scale):
...
+        >>> mesh = examples.download_bunny_coarse()
 
+        >>> pl = pv.Plotter()
+        >>> pl.add_mesh(mesh, show_edges=True, color="white")
+        >>> pl.add_points(mesh.points, color="red", point_size=10)
+        >>> print(pl.camera_position)
+        >>> pl.camera_position = [(0.02, 0.30, 0.73), (0.02, 0.03, -0.022), (-0.03, 0.94, -0.34)]
+        >>> print(pl.camera_position)
+        """
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1327]:
```suggestion
        return self.renderer.camera_position
```

File: pyvista/plotting/plotting.py [hunk 3]
```diff
@@ -1308,19 +1308,47 @@ def scale(self, scale):
...
+        
     @camera_position.setter
     def camera_position(self, camera_location):
         """Set camera position of the active render window."""
         self.renderer.camera_position = camera_location
 
     @property
     def background_color(self):
-        """Return the background color of the active render window."""
+        """Return the background color of the active render window.
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1335]:
```suggestion
        """Return the background color of the active render window."""
```

File: pyvista/plotting/plotting.py [hunk 4]
```diff
@@ -1308,19 +1308,47 @@ def scale(self, scale):
...
+
+         >>> pl = pv.Plotter()
+         >>> pl.add_mesh(mesh, show_edges=True, color="white")
+         >>> print(pl.background_color)
+         >>> pl.add_points(mesh.points, color="red", point_size=10)
+         >>> pl.background_color = "pink"
+         >>> print(pl.background_color)
+         >>> pl.show()
+        """
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1351]:
```suggestion

```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Thanks for your contribution. I fix the position of the docstring and revert the docstring of `background_color` . Could you open another Pull Request for `background_color`?

File: pyvista/plotting/plotting.py [hunk 5]
```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
...
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
+
+        Examples
+        --------
+        Return camera's position and then reposition it.
+
+        >>> mesh = examples.download_bunny_coarse()
+
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1318]: We don't need a blank line here.
```suggestion
```

File: pyvista/plotting/plotting.py [hunk 6]
```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
...
     @property
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
+
+        Examples
+        --------
+        Return camera's position and then reposition it.
+
+        >>> mesh = examples.download_bunny_coarse()
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1317]: We need to import pyvista here.
```suggestion
        >>> import pyvista as pv
        >>> from pyvista import examples
        >>> mesh = examples.download_bunny_coarse()
```

File: pyvista/plotting/plotting.py [hunk 7]
```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
...
+
+        >>> pl = pv.Plotter()
+        >>> pl.add_mesh(mesh, show_edges=True, color="white")
+        >>> pl.add_points(mesh.points, color="red", point_size=10)
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1321]: `add_mesh` and `add_points` returns `vtk.vtkActor`. We need "_" variable.
```suggestion
        >>> _ = pl.add_mesh(mesh, show_edges=True, color="white")
        >>> _ = pl.add_points(mesh.points, color="red", point_size=10)
```

File: pyvista/plotting/plotting.py [hunk 8]
```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
...
+        >>> print(pl.camera_position)
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1322]: We need to add what is returned by docstring.
```suggestion
        >>> print(pl.camera_position)
        [(0.02430187165737152, 0.033604808151721954, 0.9445935746397425),
        (0.02430187165737152, 0.033604808151721954, -0.022251706570386887),
        (0.0, 1.0, 0.0)]
```

File: pyvista/plotting/plotting.py [hunk 9]
```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
...
+        >>> pl.camera_position = [(0.02, 0.30, 0.73), (0.02, 0.03, -0.022), (-0.03, 0.94, -0.34)]
+        >>> print(pl.camera_position)
```
Comment (tkoyama010) [inline pyvista/plotting/plotting.py:1324]: We need to add what is returned by docstring.
```suggestion
        >>> print(pl.camera_position)
        [(0.02, 0.3, 0.73),
         (0.02, 0.03, -0.022),
         (-0.02999850011249063, 0.9399530035247063, -0.33998300127489384)]
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
@ENIAC-6 These are suggestions for the docstring.

**Comment by akaszynski** [comment]
I'm going to push some changes in a second. Looks like the `background_color` example has snuck in this PR.

**Comment by akaszynski** [comment]
Looks like I don't have permission to push to this. @ENIAC-6, I'm making a new PR from this, but I'll keep the core of your changes.

**Comment by akaszynski** [comment]
Closing in favor of #3019. We need a patch release for 0.35.2 and I wanted to make sure your changes went out with the next release.

**Comment by adeak** [comment]
@ENIAC-6 thanks for the PR! For future reference, when you open a PR please tick the box that says "allow edits from maintainers". See [the github docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).

This setting is necessary for us to be able to make commits to your branch, which makes collaboration more streamlined :)

The setting looks like this, below the PR's comment, next to the "Create pull request" button:
screenshot showing the position of the "allow edit from maintainers" checkbox

**Pull request closed without merging**